chunky/src/java/se/llbit/chunky/block/Block.java

```diff
@@ -58,7 +58,7 @@ public String toString() {
     return name;
   }
 
-  public boolean isBlockEntity() {
+  public boolean isBlockEntity(CompoundTag entityTag) {
```

Comment:
Hm… :thinking:
Instead of changing this signature, we could also treat this as "maybe a block entity" and allow `getBlockEntity` to return `null` if there is none.

Another (more involved) way would be to remove  `isBlockEntity` completely and move the logic into a `BlockEntityBlock` instead and check via `block instanceof BlockEntityBlock`.

I'm not sure what to do here. What do you think?

Reply:
I think the `BlockEntityBlock` solution would be a more complete solution, but yeah, it is much more involved and probably doesn't have much benefit right now as the Beacon would be the only block using that functionality. Allowing for `getBlockEntity` to return null would be much easier.

Reply:
:+1: Then go for that variant.